charts/federation-v2/values.yaml

```diff
@@ -19,18 +18,28 @@ controllermanager:
     requests:
       cpu: 100m
       memory: 64Mi
-  ## Feature gates are a set of key=value pairs that describe alpha or
-  ## experimental features. An administrator can use the --feature-gates
-  ## command line flag on each component to turn a feature on or off.
-  ## Federation V2 now have the following default feature gates:
-  ## PushReconciler=true,SchedulerPreferences=true,CrossClusterServiceDiscovery=true,FederatedIngress=true
-  featureGates:
   ## The namespace the federation control plane is deployed in. If unset,
   ## will default to federation-system.
   federationNamespace:
-  ## The cluster registry namespace. If unset, will default to kube-multicluster-public.
+  ## The cluster registry namespace.
+  ## If unset, will default to kube-multicluster-public for a cluster-scoped deployment,
+  ## and the federation namespace for namespace-scoped deployment.
   registryNamespace:
-  
+  clusterAvailableDelay:
+  clusterUnavailableDelay:
+  clusterMonitorPeriod:
+  leaderElectLeaseDuration:
+  leaderElectRenewDeadline:
+  leaderElectRetryPeriod:
+  ## Supported options are `configmaps` and `endpoints`
+  leaderElectResourceLock:
+  ## Value of feature gates item should be either `true` or `false`
+  featureGates:
```

Comment:
Given that we're not hard-coding the names of available feature gates in the API, is there a reason to hard-code them in the helm chart?

Reply:
These names in `values.yaml` exists to help users who want to enable/disable features on their own. List them here does not impact helm installation by default.

Reply:
That's fine for this PR, I'm just concerned at the prospect of hard-coding values in the helm chart that are not guaranteed to be supported in the future in the API (since `FederationConfig` exposes a list of name/value pairs).